core/src/main/java/org/elasticsearch/action/search/MultiSearchResponse.java

```diff
@@ -130,6 +134,20 @@ public MultiSearchResponse(Item[] items) {
     public Item[] getResponses() {
         return this.items;
     }
+    
+    /**
+     * How long the msearch took.
+     */
+    public TimeValue getTook() {
```

Comment:
It looks like this is only used in tests, and only to get the milliseconds, so I think that we can remove it and just use `getTookInMillis` in those places.